app/src/main/java/com/example/androidacademyhw/data/Actor.kt

```diff
@@ -1,6 +1,7 @@
 package com.example.androidacademyhw.data
 
 data class Actor(
+    val id: Int,
     val name: String,
-    val image: Int
+    val picture: String
```

Comment:
```suggestion
    val pictureUrl: String
```

Reply:
Sorry. Marked this as done because made autocommit with this suggestion.
This is fixed.